deploy/aws/README.md

```diff
@@ -1,48 +1,91 @@
 # Deploy TiDB Operator and TiDB cluster on AWS EKS
 
-## Requirements:
-* [awscli](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html) >= 1.16.73
+## Requirements
+* [awscli](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html) >= 1.16.73, to control AWS resources
+
+  The `awscli` must be [configured](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html) before it can interact with AWS. The fastest way is using the `aws configure` command:
+
+  ``` shell
+  # Replace AWS Access Key ID and AWS Secret Access Key with your own keys
+  $ aws configure
+  AWS Access Key ID [None]: AKIAIOSFODNN7EXAMPLE
+  AWS Secret Access Key [None]: wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+  Default region name [None]: us-west-2
+  Default output format [None]: json
+  ```
+  > **Note:** The access key must have at least permissions to: create VPC, create EBS, create EC2 and create role
 * [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl) >= 1.11
 * [helm](https://github.com/helm/helm/blob/master/docs/install.md#installing-the-helm-client) >= 2.9.0
 * [jq](https://stedolan.github.io/jq/download/)
-* [aws-iam-authenticator](https://github.com/kubernetes-sigs/aws-iam-authenticator) installed in `PATH`
+* [aws-iam-authenticator](https://docs.aws.amazon.com/eks/latest/userguide/install-aws-iam-authenticator.html) installed in `PATH`, to authenticate with AWS
 
-## Configure awscli
+  The easiest way to install `aws-iam-authenticator` is to download the prebuilt binary:
 
-https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html
+  ``` shell
+  # Download binary for Linux
+  curl -o aws-iam-authenticator https://amazon-eks.s3-us-west-2.amazonaws.com/1.12.7/2019-03-27/bin/linux/amd64/aws-iam-authenticator
 
-## Setup
+  # Or, download binary for macOS
+  curl -o aws-iam-authenticator https://amazon-eks.s3-us-west-2.amazonaws.com/1.12.7/2019-03-27/bin/darwin/amd64/aws-iam-authenticator
 
-The default setup will create a new VPC and a t2.micro instance as bastion machine. And EKS cluster with the following ec2 instance worker nodes:
+  chmod +x ./aws-iam-authenticator
+  sudo mv ./aws-iam-authenticator /usr/local/bin/aws-iam-authenticator
+  ```
+
+## Deploy
+
+The default setup will create a new VPC and a t2.micro instance as bastion machine, and an EKS cluster with the following ec2 instances as worker nodes:
 
 * 3 m5d.xlarge instances for PD
 * 3 i3.2xlarge instances for TiKV
 * 2 c4.4xlarge instances for TiDB
 * 1 c5.xlarge instance for monitor
 
-You can change default values in `variables.tf` (like the cluster name and versions) as needed. The default value of `cluster_name` is `my-cluster`.
-
 ``` shell
```

Comment:
Please add an introductory sentence before the shell commands to clarify what these commands do.